src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -33,28 +33,58 @@ public BookController(BookRepository bookRepository) {
         return bookRepository.findAll();
     }
 
+    /**
+     * This method consults book by author
+     *
+     * @param bookAuthor
```

Comment:
I think adding a short description of the parameter would make the javadoc more complete.